### ISSUE TITLE
need to suppress  the nginx-ingsress warning logs:  networking.k8s.io/v1beta1

### ISSUE THREAD

**Issue by vadlakiran** [nginxinc/kubernetes-ingress#3817]
**Describe the bug**

we have installed nginx-ingress version v1.10.1 and kubernetes version v1.21.5
nginx ingress image= nginx/nginx-ingress:1.10.1

nginx-ingress pod logs are giving warnings
**W0425 13:57:30.293672 1 warnings.go:70] networking.k8s.io/v1beta1 IngressClass is deprecated in v1.19+, unavailable in v
1.22+; use networking.k8s.io/v1 IngressClassList**

we want to remove/suppress the above warning so that it should not be seen in the nginx-ingress logs.
other warnings i have suppressed successfully using below code in the yaml.

spec:
containers:
- args:
- -v=2

**To Reproduce**
1. Deploy x to '...' using some.yaml
2. View logs on '....'
3. See error

**Expected behavior**

**Your environment**
* Version of the Ingress Controller - release version or a specific commit : v1.10.1
* Version of Kubernetes: v1.21.5
* Kubernetes platform (e.g. Mini-kube or GCP) : on-premises 
* Using NGINX or NGINX Plus : NGINX daemon

**Additional context**
Add any other context about the problem here. Any log files you want to share.

**Comment by brianehlert** [comment]
That warning is being emitted by the Kubernetes API / "API machinery" and not this project.
It is part of the deprecation workflow for objects.

**Comment by vadlakiran** [comment]
yeah that warnings are from kuberntes API  even we have checked all the resources if any resource using that v1beta1 but none of the resource are using.

we want to suppress from nginx-ingress daemon set logs, please help how we can suppress those warning from nginx pod ?
